Makefile

```diff
@@ -1,27 +1,29 @@
-include build.conf
 FORCE_PULL := 0
 DEV_BUILD  := 0
+ARCH       := amd64
+
+include build.conf
+include build.conf.$(ARCH)
 
 
 installer: minimal
 	docker build -t $(IMAGE_NAME):$(VERSION) .
 
-bin/rancheros:
+bin/ros:
 	mkdir -p $(dir $@)
-	go build -tags netgo -installsuffix netgo -ldflags "-X github.com/rancher/os/config.VERSION=$(VERSION) -linkmode external -extldflags -static" -o $@
-	strip --strip-all $@
+	ARCH=$(ARCH) VERSION=$(VERSION) ./scripts/mk-ros.sh $@
 
+./ros: bin/ros
+ifeq "$(ARCH)" "amd64"
+	ln -sf bin/ros ./ros
+else
+	ARCH=amd64 VERSION=$(VERSION) ./scripts/mk-ros.sh $@
```

Comment:
So, the build works in a regular docker container (on a amd64 linux server) and we need ./ros command for a later build step - extracting system container images from os-config.yml (or os-config_arm.yml). We can use the same binary when building for amd64, but need to build a separate one when building for arm.